### PR TITLE
Alignscore: better logs and fix for "state" field

### PR DESCRIPTION
**Pull request by sidravi1**
Reviewer: @suzinyou 
Estimate: 10 mins

---

## Ticket

Fixes: [AAQ-369](https://idinsight.atlassian.net/browse/AAQ-369)

## Description

Better logs when alignscore fails.
Also fixes the state field error

### Changes
- update to tests
- update to logging

## How has this been tested?
- test cases pass

## Checklist

Fill with `x` for completed. Delete any lines that are not relevant

- [x] My code follows the style guidelines of this project
- [x] I have reviewed my own code to ensure good quality
- [x] I have tested the functionality of my code to ensure it works as intended
- [x] I have resolved merge conflicts
- [x] I have updated the automated tests (if applicable)
- [x] I have updated the requirements (if applicable)
- [ ] I have updated the README file (if applicable)
- [ ] I have updated affected documentation (if applicable)

[AAQ-369]: https://idinsight.atlassian.net/browse/AAQ-369?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: core_backend/app/llm_call/check_output.py [hunk 1]
```diff
@@ -146,7 +160,7 @@ async def _get_llm_align_score(align_score_data: AlignScoreData) -> AlignmentSco
     try:
         alignment_score = AlignmentScore.model_validate_json(result)
     except ValidationError as e:
-        logger.error(f"LLM alignment score respone is not valid json: {e}")
+        logger.error(f"LLM alignment score response is not valid json: {e}")
 
     logger.info(f"LLM Alignment result: {alignment_score.model_dump_json()}")
```
Comment (suzinyou) [inline core_backend/app/llm_call/check_output.py:165]: BTW, this line just repeats the llm output log, should we delete it?
image

File: core_backend/app/llm_call/check_output.py [hunk 2]
```diff
@@ -105,8 +109,18 @@ async def _check_align_score(
     }
 
     if align_score.score < float(ALIGN_SCORE_THRESHOLD):
```
Comment (suzinyou) [inline core_backend/app/llm_call/check_output.py:111]: I am having trouble coming up with a test case for this..  RAG needs to succeed, but alignment needs to fail!

**Review by suzinyou — APPROVED** [review]
Approved.. But I could not test all logic :(

File: core_backend/app/llm_call/check_output.py (hunk 1 above)
Comment (sidravi1) [reply core_backend/app/llm_call/check_output.py:165]: I think the first one is from lite_llm logging and we may turn that off at some stage. And the second one is from us. 
My vote is to let the both stay.

File: core_backend/app/llm_call/check_output.py (hunk 2 above)
Comment (sidravi1) [reply core_backend/app/llm_call/check_output.py:111]: What if we just call `_check_align_score` with a made up `UserQueryResponse`?
It more a unit test than an end to end test.

**Pull request merged**